crates/test-utils/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "kakarot-test-utils"
+version.workspace = true
+edition.workspace = true
+authors.workspace = true
+description.workspace = true
+homepage.workspace = true
+repository.workspace = true
+readme.workspace = true
+license.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+kakarot-rpc-core={ path = "../core" }
+
+dojo-test-utils = { workspace = true }
+katana-core = { workspace = true }
+reth-primitives = { workspace = true }
+starknet = { workspace = true }
+starknet-crypto = { workspace = true }
+url = { workspace = true }
+
+lazy_static = { workspace = true }
+tokio = { workspace = true }
+
+bytes = "1"
+dotenv = { workspace = true }
+ethers = { workspace = true }
+ethers-solc = { workspace = true }
+foundry-config = { git = "https://github.com/foundry-rs/foundry", branch = "master" }
+rstest = { workspace = true }
+git2 = { workspace = true, optional = true }
+
+futures = "0.3.26"
+serde = { version = "1.0" }
```

Comment:
Can't you use serde and serde json from the workspace ?

Reply:
So, I dependencies are copied from the old toml file, but I changed them to workspace, and the tests still seem to be passing, so I have updated them via [this](https://github.com/kkrt-labs/kakarot-rpc/pull/553/commits/cafa69242c75540fa7be96a2c1b455434645d6ad) commit.